### PR TITLE
Added Insertion and Merge Sort

### PR DESCRIPTION
**Pull request by Arnab-K-S**
Added insertion and merge sort algorithms

### CODE REVIEW

**Comment by fms12** [comment]
create an issue first

File: Searching & sorting/Insertion Sort.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+def insertionSort(L):
+    arr=L
+    for i in range(1, len(arr)):
```
Comment (fms12) [inline Searching & sorting/Insertion Sort.py:3]: add comments for explanation and make a separate folder for sorting then try to push

File: Searching & sorting/Merge Sort.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+	n1 = m - l + 1
+	n2 = r - m
+
+	L = [0] * (n1)
+	R = [0] * (n2)
+
+	for i in range(0, n1):
+		L[i] = arr[l + i]
+
+	for j in range(0, n2):
```
Comment (fms12) [inline Searching & sorting/Merge Sort.py:11]: add comments first check the issue then submit PR

**Review by fms12 — CHANGES_REQUESTED** [review]

File: Searching & sorting/Insertion Sort.py (hunk 1 above)
Comment (Arnab-K-S) [reply Searching & sorting/Insertion Sort.py:3]: Okay , adding the comments.

**Review by fms12 — APPROVED** [review]

**Pull request merged**